### PR TITLE
DOC Adding missing quote on routing example

### PR DESCRIPTION
**Pull request by catharsisjelly**
## Description
Found a missing quote in the documentation for routing

## Issues
- #611 

## Pull request checklist
- [x] The target branch is correct
    - See [branches and commit messages](https://docs.silverstripe.org/en/contributing/documentation#branches-and-commit-messages)
- [x] All commits are relevant to the purpose of the PR (e.g. no TODO comments, unrelated rewording/restructuring, or arbitrary changes)
    - Small amounts of additional changes are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
- [x] The commit messages follow our [commit message guidelines](https://docs.silverstripe.org/en/contributing/code/#commit-messages)
- [x] The PR follows our [contribution guidelines](https://docs.silverstripe.org/en/contributing/documentation/)
- [x] The changes follow our [writing style guide](https://docs.silverstripe.org/en/contributing/documentation/#writing-style)
- [x] Code examples follow our [coding conventions](https://docs.silverstripe.org/en/contributing/coding_conventions/)
- [x] CI is green

### CODE REVIEW

**Comment by catharsisjelly** [comment]
A resubmit of #612 as I messed it up, this is now using the right branch. Let me know if you want any changes.

File: en/02_Developer_Guides/02_Controllers/02_Routing.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ start parsing variables and the appropriate controller action AFTER the `//`).
 ```yml
 SilverStripe\Control\Director:
   rules:
-    'admin/help//$Action/$ID: 'App\Control\AdminHelpController'
+    'admin/help//$Action/$ID:' 'App\Control\AdminHelpController'
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/02_Controllers/02_Routing.md]:
```suggestion
    'admin/help//$Action/$ID': 'App\Control\AdminHelpController'
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Thanks. Minor change - the colon is part of YAML syntax, not part of the string.

**Review by GuySartorelli — APPROVED** [review]
LGTM, thanks again

**Pull request merged**